.github/workflows/gas-report.yml

```diff
@@ -0,0 +1,54 @@
+name: Gas report
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    paths:
+      - packages/store/**
+      - packages/world/**
+
+jobs:
+  gas-report:
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+        node-version: [16.x]
+    runs-on: ${{ matrix.os }}
+
+    name: Generate gas report
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          submodules: recursive
```

Comment:
do we need this? (probably doesn't hurt, just wondering if there is a reason you added it)

Reply:
this helps if we ever move to true foundry modules